packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -1095,5 +1094,97 @@
+	"NO_USERS_TO_INVITE": "No users to invite",
+	"OWNER_ACCOUNT_BENEFITS_MESSAGE": "By setting up an owner account, you can invite other users to join your instance. It also ensures your instance can’t be accessed without an account.",
+	"PASSWORD": "Password",
+	"PASSWORDS_MUST_MATCH_ERROR": "Passwords must match",
+	"DEFAULT_PASSWORD_REQUIREMENTS": "At least 8 characters with 1 number and 1 uppercase",
+	"PASSWORD_UPDATE_ERROR": "Problem changing the password",
+	"PASSWORD_UPDATE_SUCCESS": "Password updated successfully",
+	"PASSWORD_UPDATE_SUCCESS_MESSAGE": "You can now sign in with your new password",
+	"PERSONAL": "Personal",
+	"PERSONAL_SETTINGS": "Personal Settings",
+	"PERSONAL_SETTINGS_UPDATED_SUCCESS": "Updated your details",
+	"PERSONAL_SETTINGS_UPDATE_ERROR": "Problem updating your details",
+	"RECOVERY_EMAIL_SENT": "Recovery email sent",
+	"RECOVER_PASSWORD": "Recover password",
+	"REENTER_NEW_PASSWORD": "Re-enter new password",
+	"RETURN_TO_SIGN_IN": "Back to sign in",
+	"ROLE": "Role",
+	"SAVE": "Save",
+	"SECURITY": "Security",
+	"SENDING_EMAIL_ERROR": "Problem sending email",
+	"SETTINGS": "Settings",
+	"SETTING_UP_OWNER_ERROR": "Problem setting up owner",
+	"SETUP_SMTP_TO_INVITE_USERS_MESSAGE": "Set up SMTP to invite users. <a href=\"https://docs.n8n.io/reference/user-management#smtp\" target=\"_blank\">Instructions</a>",
+	"SET_UP_ACCOUNT_ERROR": "Problem setting up your account",
+	"SET_UP_ACCOUNT_MESSAGE": "Set up your account",
+	"SET_UP_MY_ACCOUNT": "Set up my owner account",
+	"SET_UP_OWNER": "Set up owner account",
+	"SET_UP_TO_INVITE_USERS_WARNING": "Set up an owner account in order to invite other users. Once set up, each user will need to use a password to access this instance.",
+	"SIGN_IN": "Sign in",
+	"SIGN_OUT": "Sign out",
+	"SIGN_OUT_ERROR": "Could not sign out",
+	"SIGN_UP_INVITER_INFO": "{firstName} {lastName} has invited you to n8n",
+	"SKIP_OWNER_SETUP_QUESTION": "Skip owner account setup?",
+	"SKIP_SETUP": "Skip setup",
+	"SKIP_SETUP_TEMPORARILY": "Skip setup for now",
+	"SMTP_TO_ADD_USERS_WARNING": "Set up SMTP before adding users (so that n8n can send them invitation emails). <a target=\"_blank\" href=\"https://docs.n8n.io/reference/user-management#smtp\">Instructions</a>",
+	"TOKEN_VALIDATION_ERROR": "Issue validating invite token",
+	"TRANSFERRED_TO_USER": "Transferred to {user}",
+	"TRANSFER_WORKFLOWS_CREDENTIALS": "Transfer their workflows and credentials to another user",
+	"USERS": "Users",
+	"USERS_INVITED_ERROR": "Users could not be invited",
+	"USERS_INVITED_SUCCESS": "Users invited successfully",
+	"USER_DELETE_ERROR": "Problem while deleting user",
+	"USER_DELETE_SUCCESS": "User deleted successfully",
+	"USER_INVITED_ERROR": "User could not be invited",
+	"USER_INVITED_SUCCESS": "User invited successfully",
+	"USER_INVITE_ERROR": "Problem inviting user",
+	"USER_MANAGEMENT_MISSING_WARNING": "You’re missing out on user management",
```

Comment:
Elsewhere we're using `'` instead of `’`

Reply:
did a quick search. it's the opposite